index.php

```diff
@@ -212,6 +212,15 @@ function smf_main()
 	// Make sure that our scheduled tasks have been running as intended
 	check_cron();
 
+	// Privacy logic only when enabled, got no valid version, try not to call the policy data
+	if (!empty($modSettings['enable_policy_function']) && !$user_info['is_guest'])
```

Comment:
It might be a good idea to add ` && !$user_info['is_admin']` to this. Then if anything goes wrong with the privacy policy, the admin can still get to the admin centre to fix it.